File: 251008/일의 자리 배열/array-with-units.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int array[10];
    int a, b;
    int sum = 0;

    cin >> a >> b;

    array[0] = a;
    array[1] = b;

    for(int i = 2; i < 10; i++) {
        array[i] = (array[i-2]+array[i-1])%10;
    }
    for(int i = 0; i < 10; i++) {
        cout << array[i] << " ";
    }
    return 0;
}

File: 251118/일의 자리 배열/array-with-units.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int[] arr = new int[10];
        string[] input = Console.ReadLine().Split();
        arr[0] = int.Parse(input[0]);
        arr[1] = int.Parse(input[1]);
        Console.Write(arr[0] + " " + arr[1] + " ");
        for(int i = 2; i < 10; i++) {
            arr[i] = (arr[i-1]+arr[i-2])%10;
            Console.Write(arr[i] + " ");
        }
    }
}
